examples/test-site/src/components/MegaMenu/MegaMenu.token.tsx

```diff
@@ -13,74 +13,121 @@
-import { asExceptMobile } from '../Elements.token';
-import './megamenu.css';
+import { asToutHorizontal } from '@bodiless/organisms';
+import { withDesign, addClasses } from '@bodiless/fclasses';
+
 import { asToutWithPaddings, asToutDefaultStyle } from '../Tout/token';
 
-const withMegaMenuStyles = withDesign({
-  // @TODO: We add the class here to style rc-menu. Maybe can use design API if we ditch rc-menu.
-  Wrapper: addProps({ popupClassName: 'container bl-mega-menu' }),
-  // @TODO: What's the best starting width? They will shrink to fit if there are more.
-  Item: addClasses('w-1/3'),
+/**
+ * Base Menu Styles
+ * ===========================================
+ */
+const withBaseMenuStyles = withDesign({
+  Wrapper: addClasses('w-full bg-teal-600 text-white'),
+  Item: addClasses('py-1 px-3 hover:bg-teal-500 min-w-100 leading-loose text-sm'),
 });
 
-const withColumnStyles = flow(
-  withMenuSublistStyles,
+/**
+ * Base Sub Menu Styles
+ * ===========================================
+ */
+const withBaseSubMenuStyles = withDesign({
+  Wrapper: withDesign({
+    List: addClasses('bg-teal-600 text-white my-1 z-10'),
+  }),
+  Item: addClasses('py-1 px-3 hover:bg-teal-500 min-w-100 leading-loose text-sm'),
+});
+
+/**
+ * Simple Sub Menu Styles
+ * ===========================================
+ */
+const asSimpleSubMenu = flow(
+  withBaseSubMenuStyles,
 );
 
-const withBasicSubMenuStyles = flow(
-  asHorizontalSubMenu,
-  withMenuSublistStyles,
+/**
+ * Touts Sub Menu Styles
+ * ===========================================
+ */
+export const withMenuToutStyles = flow(
+  asToutWithPaddings,
+  asToutDefaultStyle,
+  asToutHorizontal,
 );
 
-const withToutSubMenuStyles = flow(
-  withBasicSubMenuStyles,
-  withMegaMenuStyles,
+const withToutStyles = withDesign({
+  Item: addClasses('w-1/3'),
+});
+
+const asToutsSubMenu = flow(
+  withToutStyles,
+  withBaseSubMenuStyles,
 );
 
-const withColumnSubMenuStyles = flow(
+/**
+ * Columns Sub Menu Styles
+ * ===========================================
+ */
+// Since removeClasses doesn't work this will allow correct hover effects on column items.
+const withColumnHoverEffect = withDesign({
+  Wrapper: withDesign({
+    WrapperItem: addClasses('hover:bg-teal-600'),
+  }),
+  Item: addClasses('hover:bg-teal-500'),
+});
+
+const withColumnStyles = flow(
   withDesign({
-    Item: withColumnStyles,
+    Item: addClasses('pr-2 pl-5'),
   }),
-  withBasicSubMenuStyles,
-  withMegaMenuStyles,
+  withColumnHoverEffect,
 );
 
-const withChameleonSubMenuStyles = withDesign({
-  Basic: withBasicSubMenuStyles,
-  Touts: withToutSubMenuStyles,
-  Columns: withColumnSubMenuStyles,
-});
-
-const withMenuStyles = flow(
+const asColumnSubMenu = flow(
   withDesign({
-    Item: withChameleonSubMenuStyles,
+    Item: withColumnStyles,
   }),
-  asHorizontalMenu,
-  withMenuListStyles,
-  asExceptMobile,
+  withBaseSubMenuStyles,
 );
 
-const withSimpleSubMenuStyles = withDesign({
-  SubMenu: withBasicSubMenuStyles,
+/**
```

Comment:
Can we split the simple menu tokens into a separate file?  If there are tokens shared between simple menu and mega menu, leave them in with the simple menu tokens and import them into the mega menu tokens, i think

Reply:
Done

Reply:
Done